README.md

```diff
@@ -364,6 +364,24 @@ handle dependencies and CLDR loading manually yourself.
 
 ## API
 
+### Bidi engine module
+
+- ** Globalize.bidiEngine( opt ) `**
+
+  Initialize Bidi engine.
+  ```javascript
+  var bidiEngine = Globalize.bidiEngine( options );
+  
+- **`bidiEng.doBidiReorder( [input]sourceText, [output, optional]sourceToTargetMap, [output, optional]levels ) `**
+
+  Returns reordered text.
+  
+  ```javascript
+  bidiEngine.doBidiReorder( "a(b)cA<B>C&123" ); 
+  // > "123&C<B>Aa(b)c" (upper case stands for Arabic or Hebrew)
+
+  [Read more...](doc/api/bidi-engine/bidi-engine.md)
+
```

Comment:
@ashensis here are some notes:

1. Headers

  this:
  ```md
  - ** Globalize.bidiEngine( opt ) `**
  ```
  should be:
  ```md
  - **`Globalize.bidiEngine( opt )`**
  ```
2. JS snippets:

  this
  ```
  ```javascript
  var bidiEngine = Globalize.bidiEngine( options );
  ```
  should be:
  <pre>
  ```javascript
  var bidiEngine = Globalize.bidiEngine( options );
  ```
  </pre>
  The same thing for the other snippet.